utils/build/docker/nodejs/express4/iast/index.js

```diff
@@ -31,107 +31,107 @@ function initMiddlewares (app) {
-  
-    var supportedAlgorithms = [ 'md5', 'sha1' ];
-  
-    var outputHashes = "";
+    const span = tracer.scope().active()
+    span.setTag('appsec.event"', true)
+
+    const supportedAlgorithms = ['md5', 'sha1']
+
+    let outputHashes = ''
     supportedAlgorithms.forEach(algorithm => {
-      var hash = crypto.createHash(algorithm).update('insecure').digest('hex')
+      const hash = crypto.createHash(algorithm).update('insecure').digest('hex')
       outputHashes += `--- ${algorithm}:${hash} ---`
-    });
-  
-    res.send(outputHashes);
-  });
-  
+    })
+
+    res.send(outputHashes)
+  })
+
   app.get('/iast/insecure_hashing/multiple_hash', (req, res) => {
-    const span = tracer.scope().active();
-    span.setTag('appsec.event"', true);
-  
-    const name = 'insecure';
-    var outputHashes = crypto.createHash('md5').update(name).digest('hex');
-    outputHashes += `--- ` + crypto.createHash('sha1').update(name).digest('hex')
-  
-    res.send(outputHashes);
-  });
-  
+    const span = tracer.scope().active()
+    span.setTag('appsec.event"', true)
+
+    const name = 'insecure'
+    let outputHashes = crypto.createHash('md5').update(name).digest('hex')
+    outputHashes += '--- ' + crypto.createHash('sha1').update(name).digest('hex')
+
+    res.send(outputHashes)
+  })
+
   app.get('/iast/insecure_hashing/test_secure_algorithm', (req, res) => {
-    const span = tracer.scope().active();
-    span.setTag('appsec.event"', true);
-  
-    res.send(crypto.createHash('sha256').update('secure').digest('hex'));
-  });
-  
-  
+    const span = tracer.scope().active()
+    span.setTag('appsec.event"', true)
+
+    res.send(crypto.createHash('sha256').update('secure').digest('hex'))
+  })
+
   app.get('/iast/insecure_hashing/test_md5_algorithm', (req, res) => {
-    const span = tracer.scope().active();
-    span.setTag('appsec.event"', true);
-  
-    res.send(crypto.createHash('md5').update('insecure').digest('hex'));
-  });
-  
+    const span = tracer.scope().active()
+    span.setTag('appsec.event"', true)
+
+    res.send(crypto.createHash('md5').update('insecure').digest('hex'))
+  })
+
   app.get('/iast/insecure_cipher/test_insecure_algorithm', (req, res) => {
-    const span = tracer.scope().active();
-    span.setTag('appsec.event"', true);
+    const span = tracer.scope().active()
+    span.setTag('appsec.event"', true)
     const cipher = crypto.createCipheriv('des-ede-cbc', '1111111111111111', 'abcdefgh')
     res.send(Buffer.concat([cipher.update('12345'), cipher.final()]))
-  });
-  
+  })
+
   app.get('/iast/insecure_cipher/test_secure_algorithm', (req, res) => {
-    const span = tracer.scope().active();
-    span.setTag('appsec.event"', true);
- 
-    const key = crypto.randomBytes(32);
- 
-    const iv = crypto.randomBytes(16);
- 
-    const cipher = crypto.createCipheriv('aes-256-cbc', Buffer.from(key), iv);
+    const span = tracer.scope().active()
+    span.setTag('appsec.event"', true)
```

Comment:
Why is there a " in the tag value `'appsec.event"'`? :S

Reply:
`appsec.event: true` is used to signal a WAF attack has been detected. Now why is it used here i have no idea haha

Reply:
it will have no effect anyway because the tests are setting `'appsec.event"'` and not `'appsec.event'`

Reply:
Oh shit I didn't even see the double quote here, that's probably not right. Which also probably means that this setTag is useless

Reply:
@uurien the git blame points at you

Reply:
wow, it seems that it is wrong, and we should remove these lines. Probably copied from java test (I checked that they are adding `appsec.event=true`, but it doesn't make any sense.
And as mentioned before... it has a typo with the last `"` ...

Reply:
removed